alibi_detect/cd/tests/test_mmd.py

```diff
@@ -3,10 +3,13 @@
 from alibi_detect.cd import MMDDrift
 from alibi_detect.cd.pytorch.mmd import MMDDriftTorch
 from alibi_detect.cd.tensorflow.mmd import MMDDriftTF
+from alibi_detect.utils.frameworks import has_keops
+if has_keops:
+    from alibi_detect.cd.keops.mmd import MMDDriftKeops
```

Comment:
Not sure we need to protect this import as tests should have all dependencies, unless the concern is that Windows/Mac tests would not run because of missing `keops` support? @ascillitoe

Reply:
Yep, exactly that. The Windows (and potentially MacOS) issues mean that we can no longer assume **all** backends are always installed during CI.